### PR TITLE
Feature/#44 fix available pot areas

### LINKED ISSUE #44: available area for PV ground exceeded by far/ sometimes inf
For some regions, the available area is exceeded by far in the "dev future" scenario.
In addition, for PV ground without agricultural areas sometimes `inf` results, because an area of 0 ha is available. 

@nesnoj could you please re-check this?

### PR DESCRIPTION
**Pull request by nesnoj**
Fixes #44

Completely reworked results for abs. and rel. areas used by RES.

### Updates
**tl;dr:** everything's fine :)

- makes use of new WEC potential areas introduced in #65 and #67 
- usable areas for small (resid) and large (ind) PV roof plants were not considered in results
- reworked calculation of potential areas in `Region.pot_areas_pv_scn()` and `Region.pot_areas_wec_scn()` which allows to pass arbitrary potential areas / land use scenario (and max. available agri area). This makes calculation of rel. areas way easier.
- **SQ scenario: Values are set to 0 as currently used areas are unknown!**
- also fixes the inf/nan stuff (set to 0)
- (some mun-specific results for ground PV didn't make sense: for the distribution of power X MW a potential area scenario is used (e.g. 'HS') considering a max. total agricultural area of Y ha (e.g. 1%=2086 ha). Let's have a deeper look on a municipality with small BAB+RAIL area but a huge AGRI area which leads to large installed power in it. If you now compare this to the available BAB+RAIL areas in analysis, you'll get very large numbers for the rel. required area. Therefore, this comparison does only make sense in highlevel results -> moved)

### Results

Rel. results per mun
- PV roof small
- PV roof large
- PV ground: current scenario
- Wind: current scenario
- Wind: legal SQ (VR/EG)

Rel. results cumulated (highlevel)
- PV roof small
- PV roof large
- PV roof cumulated
- PV ground: current scenario
- PV ground: hard only + max. 1% agri area
- PV ground: hard only + max. 2% agri area
- PV ground: hard only + max. 3% agri area
- PV ground: hard+weak + max. 1% agri area
- PV ground: hard+weak + max. 2% agri area
- PV ground: hard+weak + max. 3% agri area
- Wind: current scenario
- Wind: 1000m + no forests = legal SQ (VR/EG)
- Wind: 1000m + forests
- Wind: 500m + no forests
- Wind: 500m + forests

**I will start new runs now...**

### CODE REVIEW

**Comment by nesnoj** [comment]
@gplssm Do not spend too much time on this PR, max. check with 1 scenario or skip testing. I run all current scenarios successfully and manually checked the numbers from 3 of them.

The more interesting aspect for me: are you fine with the result list above?
